sources/admin/ManageFeatures.controller.php

```diff
@@ -82,6 +82,9 @@ class ManageFeatures_Controller extends Action_Controller
 	public function pre_dispatch()
 	{
 		$this->_req = HttpReq::instance();
+
+		// We need this in few places so it's easier to have it loaded here
+		require_once(SUBSDIR . '/ManageFeatures.subs.php');
```

Comment:
Yeah, I feel it makes sense... at some point it could be the norm to have the base subs included by default.